.github/workflows/continous-integration.yml

```diff
@@ -586,7 +571,7 @@ jobs:
             -Dsonar.organization=rasahq
             -Dsonar.projectKey=RasaHQ_rasa
             -Dsonar.sources=.
-            -Dsonar.python.coverage.reportPaths=tests_coverage/coverage.xml
+            -Dsonar.python.coverage.reportPaths=${{ github.workspace }}/tests_coverage
```

Comment:
This was the problem! How did you figure it out?

Reply:
I had a hunch it was related to the path of the coverage file + @m-vdb suggested moving the sonar step into the same step as prepare coverage which helped sonar find the folder.